Last-Update: 2012-09-24
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Forwarded: not-needed
Origin: r31877
Description: backported patch
 Start layout only after document is fully loaded

--- a/src/text/fmt/xp/fl_DocLayout.cpp
+++ b/src/text/fmt/xp/fl_DocLayout.cpp
@@ -2932,7 +2932,7 @@
 	fl_SectionLayout* pSL = m_pFirstSection;
 	while (pSL)
 	{
-	        if(!isLayoutFilling() || (pSL->getType() == FL_SECTION_DOC))
+		if(!isLayoutFilling())
 		{
 		        pSL->updateLayout(false);
 		}
